nse_nsock.cc

```diff
@@ -1104,7 +1104,8 @@ LUALIB_API int luaopen_nsock (lua_State *L)
   int top = lua_gettop(L);
 
   /* library upvalues */
-  nsock_pool nsp = new_pool(L); /* NSOCK_POOL */
```

Comment:
How is this a problem? NSE needs a Nsock pool, regardless of whether OpenSSL support is included. What's the actual warning?